plugins/org.python.pydev/src/org/python/pydev/editor/PyEdit.java

```diff
@@ -669,6 +669,9 @@ private void updateHoverBehavior() {
             String t = types[i];
 
             ISourceViewer sourceViewer = getSourceViewer();
+            if (sourceViewer == null) {
```

Comment:
Yes of course, thanks. Just pushed a change.

On Fri, May 27, 2016 at 1:12 PM, Fabio Zadrozny <notifications@github.com>
wrote:

> In plugins/org.python.pydev/src/org/python/pydev/editor/PyEdit.java
> <https://github.com/fabioz/Pydev/pull/181#discussion_r64937674>:
>
> > @@ -669,6 +669,9 @@ private void updateHoverBehavior() {
> >              String t = types[i];
> >
> >              ISourceViewer sourceViewer = getSourceViewer();
> > +            if (sourceViewer == null) {
>
> I think this should be the first thing on this method, not inside this for
> loop.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/fabioz/Pydev/pull/181/files/ea1e02c93bfae02c56cc5901d36eb87375c2678e#r64937674>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AA3OT0X0X4atTmj9-EEQxr6IYE3dRlBFks5qFyYSgaJpZM4IosE1>
> .
>